src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -42,10 +42,20 @@ abstract class AbstractFunctionalTest extends Specification {
                 .collect { "'$it'" }
                 .join(", ")
 
-        // Add the logic under test to the test build
+        // Add the logic under test to the test build. 
+        //
+        // We are adding known versions of dependencies that will break the plugin 
+        // should they leak into our custom classloader. Should our custom classloader
+        // be configured correctly adding these to the buildscript classpath should
+        // cause no issue.
         buildFile << """
             buildscript {
+                repositories {
```

Comment:
The plugin itself doesn't require an external repository on the `buildscript` level.